lib/webpacker/file_loader.rb

```diff
@@ -7,9 +7,17 @@ class FileLoaderError < StandardError; end
   attr_accessor :data
 
   class << self
+    def exist?
+      instance.data.present? rescue false
+    end
+
     def load(path = file_path)
       self.instance = new(path)
     end
+
+    def reloadable?
+      ["development", "test"].include?(Rails.env)
```

Comment:
I think environment is the wrong thing to check here exactly because of that. People may want to enable that in staging, or in an other environment. I'd introduce a new configuration like `reload_webpacker` with default to `false` and have this config set to true in the development and test environment files. Just like sprockets does with debug option for example.